stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -217,18 +220,25 @@ function editor()
 end
 
 """
-    edit(path::AbstractString, line::Integer=0, column::Integer=0)
+    edit(path::AbstractString, line::Integer=0, column::Integer=0; wait::Union{Bool, Nothing}=nothing)
 
 Edit a file or directory optionally providing a line number to edit the file at.
-Return to the `julia` prompt when you quit the editor. The editor can be changed
+
+`wait` can be `true`, `false` or `nothing`.
+
+- `wait = true`: return to the `julia` prompt when you quit the editor,
+- `wait = false`: return immediately while opening editor in a background task,
+- `wait = nothing`: default to whether to wait or not based on predefined editor callbacks.
```

Comment:
Maybe warn that true/false are not actually supported for all editors (yet)?

Reply:
Added a commit with a warning.